index.html

```diff
@@ -464,161 +820,189 @@ <h4>
-          for communication. On top of that the tag may optionally support ISO-DEP
-          (Data Exchange Protocol defined in ISO/IEC 14443 (ISO/IEC 14443-4:2008
-          Part 4: Transmission protocol). The tags are preconfigured
-          to be either rewritable or read-only. Variable memory, up to `32` kbytes.
-          Supports three different communication speeds `106` or `212` or
-          `424` kbit/s.
+          Writing to an <a>NFC tag</a> which already contains a
+          <a>NDEF message</a> with a different <a>record identifier</a>
+          (i.e. overwriting a web-specific tag).
         </li>
         <li>
-          <dfn>NFC Forum Type 5</dfn>: This tag is based on ISO/IEC 15693 (NFC-V) and
-          allows reading and writing an NDEF message on a ISO/IEC 15693 RF tag
-          that is accessible by long range RFID readers as well. The NFC communication
-          is limited to short distance and may use the <i>Active Communication Mode</i> of
-          ISO/IEC 18092 where the sending peer generates the field which balances
-          power consumption and improves link stability. Variable memory, up to `64` kbytes.
-          Communiction speed `26.48` kbit/s
+          Writing to an <a>NFC tag</a> which already contains a
+          <a>NDEF message</a> with the same <a>record identifier</a>
+          (i.e. updating own tag).
         </li>
         <li>
-          <dfn>MIFARE Standard</dfn>: This tag, often sold under the brand names MIFARE
-          Classic or MIFARE Mini, is based on the ISO/IEC 14443-3A (also known as NFC-A,
-          as defined in ISO/IEC 14443-3:2011, Part 3: Initialization and anticollision).
-          The tags are rewritable and can be configured to become read-only. Memory size
-          can be between `320` and `4` kbytes. Communication speed is `106` kbit/sec.
-          <p class=note>
-            <a>MIFARE Standard</a> is a not an NFC Forum type and can only be read by devices
-            using NXP hardware. Support for reading and writing to tags based on the
-            <a>MIFARE Standard</a> is thus non-nominative, but the type is included
-            due to the popularity and use in legacy systems.
-          </p>
+          Writing to other, writable <a>NFC tag</a>s (i.e. overwriting a
+          generic tag).
         </li>
       </ol>
     </p>
+    <p class="note">
+      Note that an NFC write operation to an <a>NFC tag</a> always involves
+      also a read operation.
+    </p>
+  </section>
+  <section> <h3>Pushing data to an <a>NFC peer</a> device</h3>
     <p>
-      In addition to data types standardized for <a>NDEF record</a>s by the NFC
-      Forum, many commercial products such as bus cards, door openers may be based
-      on the <a>MIFARE Standard</a> which require specific NFC chips (same vendor of
-      card and reader) in order to function.
+      In general, pushing data to another Web NFC capable device requires that
```

Comment:
remove Web, it might not even run a web browser

Reply:
done